x-pack/platform/plugins/shared/streams/server/routes/streams/crud/read_stream.ts

```diff
@@ -139,7 +150,12 @@ export async function readStream({
     effective_lifecycle: findInheritedLifecycle(streamDefinition, ancestors),
     effective_settings: getInheritedSettings([...ancestors, streamDefinition]),
     inherited_fields: inheritedFields,
+    effective_failure_store: failureStore
+      ? {
+          ...failureStore,
+          from: inheritedFailureStore.from,
+        }
+      : undefined,
```

Comment:
This should always be defined. Could we assign the `inheritedFailureStore` here, like we do for lifecycle, fields and settings ?